deployments/container/Dockerfile.ubi8

```
@@ -14,7 +14,7 @@
 
 ARG CUDA_VERSION
 ARG BASE_DIST
-ARG GOLANG_VERSION=1.20.2
+ARG GOLANG_VERSION=1.20.5
```

Let's use:
```suggestion
ARG GOLANG_VERSION=x.x.x
```
here so that we don't have to update it (and keep it in sync) when bumping the golang version.